### PR TITLE
Add owlracle api

### PR DESCRIPTION
**Pull request by groninge01**
Adds the owlracle gas api into the backend.

### CODE REVIEW

**Comment by groninge01** [comment]
Still working on it...

**Comment by groninge01** [comment]
@danielmkm This PR has been updated and is ready for review now.

**Comment by groninge01** [comment]
Fixes #8.

File: app/env.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export const schema = {
     COPPER_PROXY_ADDRESS: String,
     RPC_URL: String,
     YEARN_VAULTS_ENDPOINT: String,
+    OWLRACLE_API_KEY: String,
```
Comment (danielmkm) [inline app/env.ts:33]: @franzns can you make sure you add the api key to all deployment

File: modules/gas-estimate/lib/owlracle.service.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    percentile: number;
+    accept: string;
+    version: number;
+}
+export class OwlracleService {
+    private readonly baseUrl: string;
+    private readonly requestParams: OwlracleRequest;
+
+    constructor() {
+        this.baseUrl = 'https://owlracle.info/ftm/gas';
```
Comment (danielmkm) [inline modules/gas-estimate/lib/owlracle.service.ts:18]: make this url configurable, ie: what if this needed to work for a different chain?

File: app/env.ts (hunk 1 above)
Comment (franzns) [reply app/env.ts:33]: I can do that, if you can get it to me :)

Comment (franzns) [reply app/env.ts:33]: env var is configured everywhere

File: modules/gas-estimate/lib/owlracle.service.ts (hunk 2 above)
Comment (groninge01) [reply modules/gas-estimate/lib/owlracle.service.ts]: @danielmkm I have updated the PR to make it multi chain. If it's ok I'll need to update the FE implementation too.

**Comment by 0xSkly** [comment]
closed due to inactivity, we can bring it back later

**Pull request closed without merging**